features/metadata.feature

```diff
@@ -51,37 +50,27 @@ Feature: Metadata
         And the "BAZ" job is started
         Then in the build, the { "foo": <foobar> } is available from metadata
         And in the build, the { "bar": <barbaz> } is available from metadata
+        And the build succeeded
+        And the event is done
+        Then a record of the metadata is stored
 
         Examples:
             | foobar       | barbaz       |
-            | "Foo"        | "Bar"        |
+            | "foobar"     | "barbaz"     |
```

Comment:
Is there a problem with changing to "foobar" from "FOO". In [functional-metadata](https://github.com/screwdriver-cd-test/functional-metadata/commit/6e2f1dfee80e94c20de9fc6b294dfd2027909542) I described it as "foobar".